content/docs/sidebar.json

```diff
@@ -74,8 +74,8 @@
         "label": "Sharing Data and Model Files",
         "slug": "sharing-data-and-model-files"
       },
-      "shared-development-server",
-      "data-registries"
+      "data-registries",
+      "shared-development-server"
```

Comment:
I feel like Data Registries has become quite popular so the shared dev env seems like more of an edge case now?

Reply:
👍